website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,77 @@
 ---
 title: "Job notifications"
 id: "job-notifications"
-description: "Set up notifications in dbt Cloud to receive Email or Slack alerts for job run status."
+description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
+Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
 
-Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+## Email notifications
 
-### Email
+You can receive email alerts by configuring the dbt Cloud notification settings. As a **user**, you can set up email notifications for yourself. As an **admin**, you can set up notifications for yourself and other team members. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for details on license types that are eligible for email notifications.
 
-These are the following options for setting up email notifications. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for info on license types eligible for email notifications. 
+### Configure email notifications
 
-- As a **user** &mdash; You can set up email notifications for yourself under your Profile.
-- As an **admin** &mdash; You can set up notifications on behalf of your team members. 
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. By default, the **Notification email** option is set to the email address that's in [your dbt Cloud personal profile](https://cloud.getdbt.com/settings/profile). 
 
-To set up job notifications, follow these steps:
+    If you're an **admin**, you can choose a different email address. Select the **Notification email** dropdown and choose another address from the list. The list includes internal users with access to the account and also external emails that have been added. 
+    - To add an external email address, select the **Notification email** dropdown and choose **Add external email**. After you add the external email, it becomes available for selection in the **Notification email** dropdown list.
```

Comment:
Can we provide some examples of when this might be useful? For example something like

While external email notifications can be used to notify people outside of your dbt Cloud account, they can also be used to setup third party integrations that are not yet natively supported. For example:
- Send notifications to microsoft teams channels: https://support.microsoft.com/en-us/office/tip-send-email-to-a-channel-2c17dbae-acdf-4209-a761-b463bdaaa4ca#:~:text=Select%20More%20options,address%20into%20a%20new%20email.
- Send notifications to pagerduty: https://support.pagerduty.com/docs/email-integration-guide